stock_measuring_device_zippcube/models/zippcube_device.py

```diff
@@ -0,0 +1,109 @@
+# Copyright 2021 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+import logging
+
+from odoo import _, fields, models
+from odoo.exceptions import UserError
+
+_logger = logging.getLogger(__name__)
+
+
+class ZippcubeDevice(models.Model):
+    _name = "zippcube.device"
+    _description = "Measuring Device Zippcube"
+    _order = "warehouse_id, name"
+
+    name = fields.Char("Name", required=True)
+    warehouse_id = fields.Many2one("stock.warehouse", "Warehouse")
+
+    _sql_constraints = [
+        (
+            "name_uniq",
+            "unique (name)",
+            "The name of the zippcube device must be unique.",
+        ),
+    ]
+
+    def open_wizard(self):
+        self.ensure_one()
+        return {
+            "name": _("Zippcube Wizard"),
+            "res_model": "zippcube.wizard",
+            "type": "ir.actions.act_window",
+            "view_id": False,
+            "view_mode": "form",
+            "context": {"default_device_id": self.id},
+            "target": "fullscreen",
+            "flags": {
+                "withControlPanel": False,
+                "form_view_initial_mode": "edit",
+                "no_breadcrumbs": True,
+            },
+        }
+
+    def _is_being_used(self):
+        """Returns whether the device is in use
+
+        A device is in use if it has been assigned to a product packaging."""
+        return bool(
+            self.env["product.packaging"].search(
+                [("scan_device_id", "=", self.id)], count=True, limit=1
```

Comment:
`limit` is useless if you use `count` no?

Reply:
You are right... I change it with `search_count` then. Thanks. (My idea here was to check just for existence, but `LIMIT` doesn't stop scanning).